database/wiiu/accept_friend_request_and_return_friend_info.go

```diff
@@ -97,11 +96,8 @@ func AcceptFriendRequestAndReturnFriendInfo(friendRequestID uint64) *friends_wii
 		if err != nil {
 			lastOnlineTime = nex.NewDateTime(0).Now()
 
-			if err == gocql.ErrNotFound {
-				globals.Logger.Error(err.Error())
-			} else {
-				globals.Logger.Critical(err.Error())
-			}
+			// TODO: Should we return the error here?
```

Comment:
Yes this should return an error. I can't remember why this is defaulted like this, but now whenever a user connects their user data is updated in `RegisterEx`. The only time this should ever error is if the data can't be found for the user in the friend request, which is critical tbh